vm_create_image.yml

```diff
@@ -15,7 +15,7 @@
     vm_name: testvm
     vm_image_name: testvmimage
     location: eastus
-    admin_username: zim
+    admin_username: azureuser
```

Comment:
no! no! no! ;-)

Reply:
you can change your display name to azureuser :)